### DISCUSSION TITLE
Error opening map? 🤔

### DISCUSSION THREAD

**Discussion by riqzor** [FacilMap/facilmap#236] [Q&A]
Hey,

I've created a map some time ago: facilmap.org/tspeaking
When I try to open the map Facilmap says "Error opening map" "getaddrinfo EAI_AGAIN mysql"

What could be wrong with it? I have a couple of hundred points saved there 😬

Thanks a lot for your help in advance!

All the best,
- Riqzor

**Comment by Droidphilev** [comment]
Same for me. Hopefully no data will be lost.

**Comment by cdauth** [comment]
Sorry for the inconvenience, the hard disk was full, causing the mysql server to crash. I have freed some space, it should work again now.

**Reply by Droidphilev** [reply]
MANY thanks!!! Working again, pfew :-)
